app/src/main/res/layout/fragment_reviews.xml

```diff
@@ -0,0 +1,13 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <android.support.v7.widget.RecyclerView
```

Comment:
If it's the exact same layout like `fragment_ingredients.xml`, I think you can probably reuse it with something like: `<include layout="@layout/fragment_ingredients" />`